packages/transaction-controller/src/utils/validation.test.ts

```diff
@@ -187,5 +188,137 @@ describe('validation', () => {
         } as any),
       ).not.toThrow();
     });
+
+    it('throws if gasPrice is defined but type is feeMarket', () => {
```

Comment:
As there is many tests here, is it worth a simple `describe` block such as `gas fees`?

Reply:
Added block in https://github.com/MetaMask/core/pull/1978/commits/c4256ce190e26145f1949211b2c4a0a271e1d3cd